### PR TITLE
feat: image zoom

### LINKED ISSUE #1084: [Site]: Add lightbox to display images
### Current State

**Description of the current state/issue**:

Currently, to zoom in to an image, users have to open just the image in a new tab.

This makes it difficult to understand large diagrams.

### Desired State

**Description of the change you are proposing**:

Instead, add a lightbox feature that would open up a modal that the users can interact with when clicking an image.

Here is an example of a lightbox: https://www.lightgalleryjs.com/

### Would you like to fix this issue?

No. Please assign another contributor

### PR DESCRIPTION
**Pull request by SkyeYoung**
Fixes: #1084 

Changes:

- add image zoom box
- reformat css

Screenshots of the change:

### CODE REVIEW

**Comment by SkyeYoung** [comment]
cc @juzhiyuan @navendu-pottekkat

**Comment by SkyeYoung** [comment]
The image in https://deploy-preview-1118--apache-apisix.netlify.app/zh/blog/2022/03/02/zhongan-usercase-with-apache-apisix looks not good, so if the background of the image is transparent...
image
image

File: website/src/theme/BlogPostPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      pageClassName={ThemeClassNames.page.blogPostPage}
+      sidebar={sidebar}
+      toc={
+        !hideTableOfContents && BlogPostContents.toc
+          ? BlogPostContents.toc
+          : undefined
+      }
+    >
+      <Seo
+        // TODO refactor needed: it's a bit annoying but Seo MUST be inside BlogLayout
```
Comment (juzhiyuan) [inline website/src/theme/BlogPostPage/index.tsx:55]: @SkyeYoung How about adding an issue to track?

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

File: website/src/theme/BlogPostPage/index.tsx (hunk 1 above)
Comment (SkyeYoung) [reply website/src/theme/BlogPostPage/index.tsx:55]: > @SkyeYoung How about adding an issue to track?

It's created by docusaurus...

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**